### DISCUSSION TITLE
Deploy vs. Install?

### DISCUSSION THREAD

**Discussion by boozedog** [dnnsoftware/Dnn.Platform#5781] [Q&A] [answered]
On the releases page, I see a `_Deploy` as well as an `_Install` asset ... where can I find documentation on the difference?

I looked at https://docs.dnncommunity.org/content/tutorials/server/setup/administrators-setup-overview/index.html but there's no mention of the word "deploy" there

Cross-posted at https://dnncommunity.org/forums/Getting-Started/installing-dnn/deploy-vs-install/

**Comment by mitchelsellers** [comment, marked as answer]
The "Deploy" package is in a format for use with some automated deployment processes.  

@dnnsoftware/approvers do we even need to distribute the `_Deploy` asset anymore?

**Comment by valadas** [comment]
I have never used this is...
- A WebDeploy package, in such case it could be useful for people on Azure and such to install DNN ?
- A package meant for Web Platform Installer, which would have been useful for some hosting control panels automations, but which is deprecated, see https://blogs.iis.net/iisteam/web-platform-installer-end-of-support-feed

**Comment by sleupold** [comment]
The deploy format was used to submit DNN to the Microsoft Web Application Gallery, which allowed automatic provisioning in hosting panels like WebSitePanel / SolidCP. 
The Microsoft Application Gallery has been terminated in 2021, I am not aware, whether the format is still used for providing applications in other tools.

**Comment by valadas** [comment]
It is technically still supported in SolidCP (which I use) but you need to create your own feed as the Microsoft one was pulled out. I believe the same is true for Plesk but I am not 100% sure... I could see still a use case for this but would not cry if that zip gets removed from our distribution either. I am neutral :)

**Comment by david-poindexter** [comment]
`DotNetNuke` still shows up in the Plesk "Application Vault".  However, I believe this is based on "Application Packaging Standard (APS)".  I am pretty sure this is not our "Deploy" package per se.  Mostly likely somebody somewhere is creating an APS for DNN.  ;-)

<img width="625" alt="image" src="https://github.com/dnnsoftware/Dnn.Platform/assets/4568451/3606854e-a331-4edd-b94c-42d9ddc63e90">

**Comment by bdukes** [comment]
I don't see a need for us to continue to provide the deploy artifact
